File: CodeForces/Java/CF_144A_ArrivaloftheGeneral.java
// Problem: 144A. Arrival of the General
// Link: https://codeforces.com/problemset/problem/144/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int n = scan.nextInt();
    
    int maxVal = 0;
    int minVal = 101;
    int maxIndex = 0;
    int minIndex = 0;
    
    for (int i = 0; i < n; i++) {
      int height = scan.nextInt();
      
      if (height > maxVal) {
        maxVal = height;
        maxIndex = i;
      }
      
      if (height <= minVal) {
        minVal = height;
        minIndex = i;
      }
    }
    
    scan.close();
    
    if (maxIndex > minIndex) {
      System.out.println(maxIndex + (n - 1 - minIndex) - 1);
    } else {
      System.out.println(maxIndex + (n - 1 - minIndex));
    }
  }
}

File: CodeForces/Python/CF_144A_ArrivaloftheGeneral.py
# Problem: 144A. Arrival of the General
# Link: https://codeforces.com/problemset/problem/144/A

n = int(input())
heights = list(map(int, input().split()))

max_val = 0
min_val = 101
max_index = 0
min_index = 0

for i in range(n):
    if heights[i] > max_val:
        max_val = heights[i]
        max_index = i
        
    if heights[i] <= min_val:
        min_val = heights[i]
        min_index = i

if max_index > min_index:
    print(max_index + (n - 1 - min_index) - 1)
else:
    print(max_index + (n - 1 - min_index))
